code/HabitTracker/app/src/main/java/com/example/habittracker/activities/eventlist/EventDetailActivity.java

```diff
@@ -66,6 +66,7 @@
     private Habit habit;
     private static final int PICK_IMAGE_REQUEST = 1;
     private static final int TAKE_PHOTO_REQUEST = 2;
+    private static final int SELECT_LOCATION_REQUEST = 3;
```

Comment:
You have to change the result code in map activity as well. This won't work until you changed the result code in MapActivity.